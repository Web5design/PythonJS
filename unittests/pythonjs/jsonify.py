print pythonjs_to_json(dict(a=1, b=2))
print pythonjs_to_json(dict(a=1.2, b=undefined))
a = list()
a.append("spam")
a.append("egg")
print pythonjs_to_json(a)
print pythonjs_to_json(dict(a_list=a))
