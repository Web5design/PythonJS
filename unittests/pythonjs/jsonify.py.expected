{"a":1,"b":2}
{"a":1.2}
["spam","egg"]
{"a_list":["spam","egg"]}
